api/envoy/config/filter/udp/udp_proxy/v2alpha/udp_proxy.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.config.filter.udp.udp_proxy.v2alpha;
+
+option java_outer_classname = "UdpProxyProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.filter.udp.udp_proxy.v2alpha";
+
+import "google/protobuf/duration.proto";
+
+import "validate/validate.proto";
+
+// TODO(mattklein123): docs
+
+message UdpProxyConfig {
+  oneof cluster_specifier {
+    option (validate.required) = true;
+
+    // The upstream cluster to connect to.
+    string cluster = 1 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
Will we eventually set timeouts here?

Reply:
Yes I have these in a follow up. I'm trying to keep this as simple as possible. See the TODOs in the filter.